docs/devel/networking.md

```diff
@@ -205,3 +205,124 @@ to the user.
 When the pod networking interface does not feature an IP address, the in-pod
 DHCP server will not be started, leaving the VM with plain L2 connection via
 the in-pod bridge.
+
+### Masquerade binding mechanism
+Similar to the [bridge bind mechanism](#bridge-binding-mechanism), triggering
+the masquerade `BindMechanism` requires a VMI configuration featuring a
+network with the `masquerade` interface type. KubeVirt provides an example of
+a
+[fedora based VMI](https://github.com/kubevirt/kubevirt/blob/master/examples/vmi-masquerade.yaml)
+in the project's examples folder.
+
+The masquerade bind mechanism has plenty in common with bridge binding; both
+have virt-handler create an in-pod bridge, set the pod networking interface as
```

Comment:
Sorry for not reviewing this on time. But this is not correct. While in the bridge binding the pod interface is a slave of the bridge. In masquerade it is not the case.

In masquerade we are creating a dummy nic called 'k6t-nic'. This nic is the first slave of the bridge (it is needed since the bridge gets the mac address of its first slave and we don't want it to take the mac address of the first tap device attached to it).